cli/viam/main.go

```diff
@@ -765,6 +758,10 @@ func main() {
 							{
 								Name:  "shell",
 								Usage: "start a shell on a robot part",
+								// TODO: remove this warning
+								Description: `Functionality of the shell command is highly experimental. In particular, there may be text-input issues
```

Comment:
Opt: If desired, such notices are often prepended with a direct warning:

```suggestion
								Description: `WARNING: Functionality of the shell command is highly experimental. In particular, there may be text-input issues
```

Reply:
Good point; added.